packages/perspective-jupyterlab/src/config/plugin.config.js

```diff
@@ -25,11 +25,45 @@ module.exports = {
     plugins: [new webpack.ContextReplacementPlugin(/moment[\/\\]locale$/, /(en|es|fr)$/), new PerspectivePlugin()],
     module: {
         rules: [
+            {
```

Comment:
We should not remove the `less` loaders here.  Note there is no `style-loader` - these are not CSS asset files, they are [inline styles for the shadow DOM](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_shadow_DOM#Attaching_the_shadow_DOM_to_the_shadow_root).  They cannot by inlined as `<style>` tags in `<head>`, and need to be present as JS strings during component registration, so a standard webpack config which also uses `style-loader` will cause these imports to fail cryptically.

Reply:
> We should not remove the less loaders here. 

Is this a typo? Can you clarify what you think the code should be here is?